data/formats-data.js

```diff
@@ -1453,7 +1453,7 @@ exports.BattleFormatsData = {
 		tier: "Uber"
 	},
 	mewtwomegax: {
-		randomBattleMoves: ["bulkup","drainpunch","earthquake","taunt","stoneedge","zenheadbutt","icebeam"],
+		randomBattleMoves: ["bulkup","drainpunch","earthquake","taunt","stoneedge","zenheadbutt","icepunch"],
```

Comment:
It prefers Ice Beam for non-setup and does not need Ice Punch for setup. Revert this.